eng/scripts/CodeChecks.ps1

```diff
@@ -16,6 +16,7 @@ param (
 )
 
 $ErrorActionPreference = 'Stop'
+$Env:NODE_OPTIONS = "--max-old-space-size=8192"
```

Comment:
This is to fix  the out-of-memory error in nodejs vm during autorest execution. There is a similar fix in https://github.com/Azure/azure-sdk-for-net/blob/03e7ee99d727338fe0b28bd9b7a874ffb5c4b5c6/eng/scripts/Track1CodeCheck.ps1#L2